input/resources/StructureDefinition-profile-SignalUSCoreOrganizationProfile.json

```diff
@@ -53,6 +63,28 @@
         ],
         "isModifier": false
       },
+      {
+        "id": "Organization.identifier:EIN",
+        "path": "Organization.identifier",
+        "sliceName": "EIN",
+        "short": "Employer Identification Number (EIN)",
+        "definition": "An Employer Identification Number (EIN) is also known as a Federal Tax Identification Number, and is used to identify a business entity. Generally, businesses need an EIN. Provided by the US Internal Revenue Service (IRS)",
+        "comment": "An Employer Identification Number (EIN), also known as a Federal Tax Identification Number, is used to identify a business entity.",
+        "patternIdentifier": {
+          "system": "urn:oid:2.16.840.1.113883.4.4"
+        }
+      },
+      {
+        "id": "Organization.identifier:MMIS",
+        "path": "Organization.identifier",
+        "sliceName": "MMIS",
+        "short": "Medicaid Management Information System (MMIS)",
+        "definition": "Medicaid Management Information System (MMIS) is the official title of Colorado's patient and provider processing network.",
+        "comment": "The Colorado MMIS, interChange, is currently operated by Gainwell Technologies. Learn more at https://hcpf.colorado.gov/mmis-procurement",
+        "patternIdentifier": {
+          "system": "https://signalbhn.org//identifiers/organization/"
```

Comment:
Sorry, let's change this system to `https://signalbhn.org//identifiers/organization-mmis/` in case we have any new ones in the future.

Reply:
Modified the system URI. #https://github.com/enjoysparkling/signal-mso-fhir-profiles/pull/113/commits/3046f47191ea43f6485961f3ad1b7dba01077e0e